models/userSchema.js

```diff
@@ -0,0 +1,70 @@
+const { checkSchema } = require('express-validator');
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  checkSchema(
+    {
+      name: {
+        type: String,
+        required: true,
+        maxLength: 22,
+      },
+      email: {
+        type: String,
+        required: true,
+        unique: true,
+        trim: true,
+        lowercase: true,
+        maxLength: 20,
+        isEmail: {
+          bail: true,
```

Comment:
what does this **bail** do.?

Reply:
Returns: the current validation chain instance. Stops running validations if any of the previous ones have failed. Useful to prevent a custom validator that touches a database or external API from running when you know it will fail.